ide-laf-bridge/src/main/kotlin/org/jetbrains/jewel/bridge/theme/IntUiBridge.kt

```diff
@@ -966,10 +968,11 @@ private fun readTextFieldStyle(): TextFieldStyle {
         colors = colors,
         metrics =
             TextFieldMetrics(
-                cornerSize = CornerSize(DarculaUIUtil.COMPONENT_ARC.dp / 2),
-                contentPadding = PaddingValues(horizontal = 9.dp, vertical = 2.dp),
+                cornerSize = retrieveArcAsCornerSize("Component.arc"),
+                contentPadding = PaddingValues(horizontal = 8.dp, vertical = 0.dp),
```

Comment:
You can omit the `0.dp` parameter:

```suggestion
                contentPadding = PaddingValues(horizontal = 8.dp),
```

Reply:
Done in 26c91bf7